scripts/docker_cmake

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+rm -f  CMakeCache.txt
+rm -rf CMakeFiles/
```

Comment:
I understand your point. The problem is that at some point you'll start rebuilding in the same directory with a changed environment, and that would lead to subtle errors. So one would always have to remember to clean up after environment changed. I got hit by these issues too many times so I simply don't trust myself to do that. There really is not a great solution here except "rm -rf" in conjunction with ccache.